FrontEnd/src/components/SignUp/UI/dotDecor/DotDecor.jsx

```diff
@@ -1,3 +1,5 @@
+import React from 'react';
```

Comment:
From React 17 there is no need to import React into the component. If you need to use `Fragment` you can import only it. 
`import { Fragment } from 'react';`

Reply:
Done